ambry-store/src/main/java/com.github.ambry.store/BlobStoreCompactor.java

```diff
@@ -717,6 +723,28 @@ private boolean alreadyExistsInTgt(StoreKey key, IndexValue srcValue) throws Sto
         IndexValue.Flags.Delete_Index));
   }
 
+  /**
+   * Calculate the farthest index that can be used for a bundle IO read based on startIndex and bundleReadBuffer capacity.
+   * @param srcIndexEntries all available index entries.
+   * @param startIndex the startIndex for a bundle read.
+   * @return the endIndex which can be used for a bundle read.
+   */
+  private int getBundleReadEndIndex(List<IndexEntry> srcIndexEntries, int startIndex) {
+    long startOffset = srcIndexEntries.get(startIndex).getValue().getOffset().getOffset();
+    int endIndex;
+    // [startIndex, endIndex) for a bundle of read.
```

Comment:
This comment is not very meaningful. Particularly, since elsewhere `endIndex` seems to be inclusive for bundled read.